### DISCUSSION TITLE
Unable to login

### DISCUSSION THREAD

**Discussion by dineshkathir** [leikoilja/ha-google-home#539] [Q&A] [answered]
I tried https://myaccount.google.com/apppasswords and also tried turning off the 2FA of my account. But still, I am unable to log in to my account.  

Kindly guide me on this.

<img width="407" alt="ghha" src="https://user-images.githubusercontent.com/69247666/171127660-dd3cf624-2afb-4829-a1c6-5a8d738a748d.png">

**Comment by leikoilja** [comment, marked as answer]
This issue is know and confirmed. It's currently being addressed at https://github.com/leikoilja/glocaltokens/issues/245. Please subscribe to that issue to keep up to date

**Reply by leikoilja** [reply]
Fixed in `ha-google-home` v1.9.11 🚀
https://github.com/leikoilja/ha-google-home/releases/tag/v1.9.11

Please update to the latest version and verify it works :)
